### PR TITLE
Use latest black version

### LINKED ISSUE #122: Black pre-commit unstable
Back pre-commit it is unstable.

When I run `docker-compose run --rm processing black . --check`, I get
```
(...)
would reformat /mnt/code/tests/gazette/data/test_bidding_exemption_parsing.py
All done! ✨ 🍰 ✨
1 file would be reformatted, 56 files would be left unchanged.
```
So I run `docker-compose run --rm processing black . --safe` and this file is formatted.

But when I try to commit, pre-commit prevent and format the file back to the format that black from docker complained about it (test_bidding_exemption_parsing).

I suspect that the black inside the docker and the black used in the pre-commit are different, but I do not know how to turn them equal

----------
I already tried to: 
- Install black globally in the same version of requirements.txt.
- Use the `entry` argument on `.pre-commit-config.yaml` using the docker command of black.

### PR DESCRIPTION
**Pull request by giovanisleite**
Fix #122

### CODE REVIEW

File: processing/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-black==18.5b1
+black
```
Comment (cuchi) [inline processing/requirements.txt]: I would use fixed versions for that kind of dependency.
```suggestion
black==18.9b0
```

Comment (giovanisleite) [reply processing/requirements.txt]: With fixed version, we break the pre-commit. We need the pre-commit with a fixed version (but reading [here](https://github.com/ambv/black#version-control-integration), I understood that we have only two options. Use the last stable version (my change) or the master branch (it is not what we want.)
black is a dependency about code-style, so it don't affect the operation, just the code itself.

Comment (giovanisleite) [reply processing/requirements.txt]: The way that it is today (fixed version), we will have problems like #122. Local black considers the code ok, pre-commit doesn't and fix it... forever.

Comment (cuducos) [reply processing/requirements.txt]: Isn't it an option to fix `black` and `pre-commit` versions to something that actually work together?

Comment (giovanisleite) [reply processing/requirements.txt]: I thought that there was just two options to rev: `stable` or `master`, but it worked.

**Comment by giovanisleite** [comment]
@cuchi @cuducos It looks like they're agreeing with each other now

**Pull request merged**